front/components/assistant_builder/InstructionScreen.tsx

```diff
@@ -403,9 +398,9 @@ function AdvancedSettings({
               </DropdownMenuTrigger>
               <DropdownMenuContent>
                 <DropdownMenuLabel label="Best performing models" />
-                <ScrollArea className="max-h-[300px]">
+                <ScrollArea className="max-h-[300px] overflow-y-scroll">
```

Comment:
`nit`: could we do `<ScrollArea className="h-[300px]">`?
Otherwise we will get two scrollbars I'm afraid

Reply:
The ScrollArea needs a fixed height to work properly, which is why the `max-h-[300px]` alone wasn't working properly

Reply:
Tried on my local and seems to do the job 👌🏼